### ISSUE TITLE
Question: How long is the battery life?

### ISSUE THREAD

**Issue by aymane-eljerari** [carrefinho/forager#2]
I'm excited to build this, and im wondering how long the battery lasts. What's the largest battery I could fit in the case?

**Comment by carrefinho** [comment]
One to two weeks on the left side (central), two to three months on the right side (peripheral). It's designed around the commonly available 110mAh 301230 battery, and because of the compact footprint, there is only a couple millimeters of extra room.

**Comment by lukeberry99** [comment]
Out of curiosity, would this work having a third microcontroller as a “dongle” that is permanently plugged in, giving both halves two to three months battery life?

**Comment by carrefinho** [comment]
@lukeberry99 
Yes, that is an option. There is [work-in-progress documentation in the ZMK docs](https://deploy-preview-2525--zmk.netlify.app/docs/development/hardware-integration/dongle).
Also check out [Prospector](https://github.com/carrefinho/prospector) for a fancy dongle.

**Comment by lightningspirit** [comment]
@carrefinho Would a LiPo 301430 (32x14x4) which is 150 mAh fit in this case?

**Comment by lightningspirit** [comment]
> Would a LiPo 301430 (32x14x4) which is 150 mAh fit in this case?

If anyone is wondering, I was able to use this battery in the case successfully.
